cloudant-sync-datastore-android-encryption/src/main/java/com/cloudant/sync/internal/sqlite/android/AndroidSQLCipherSQLite.java

```diff
@@ -101,19 +102,27 @@ public void setTransactionSuccessful() {
     }
 
     @Override
-    public void execSQL(String sql)  {
+    public void execSQL(String sql) throws java.sql.SQLException {
```

Comment:
Good spot. Fixed in 4272ecb.

I've handled it the same way as the java se version in [SQLiteWrapper.java](https://github.com/cloudant/sync-android/blob/master/cloudant-sync-datastore-javase/src/main/java/com/cloudant/sync/sqlite/sqlite4java/SQLiteWrapper.java#L104) so they are consistent.